adapters/dmx/dmx.go

```diff
@@ -55,6 +64,7 @@ func (adapter *DmxAdapter) MakeRequests(request *openrtb.BidRequest, req *adapte
 	var anyHasId = false
 	var reqCopy openrtb.BidRequest = *request
```

Comment:
More an FYI, I don't expect you to act on this in this PR. The request being passed to an adapter is a shallow copy of the original request that is unique to your adapter, so you don't need to make another shallow copy of it. But nice to see attention to insuring we don't disrupt shared memory structures.

Reply:
Will update this in my next PR